### DISCUSSION TITLE
What are the best settings you found ?

### DISCUSSION THREAD

**Discussion by bobymadoff** [EA31337/EA31337#407] [Q&A]
What are the best settings you have found for the advanced, lite, rider and elite EA ?

**Comment by kenorb** [comment]
Before each release/version, EA is being heavily optimized and the default parameters are usually set for the most optimal which can work for EURUSD at that time of release. So using the default parameters are recommended in long term, however you can still try to optimize it further adjusting to your needs and individual setup (broker, spread, leverage, risk and so on).
